docker/recorder/src/main/java/org/arquillian/cube/docker/impl/client/reporter/TakeDockerEnvironment.java

```diff
@@ -189,8 +279,133 @@ private File createSchemasDirectory(File rootDirectory) {
         return schemasDir.toFile();
     }
 
+    private File createContainerLogDirectory(File rootDirectory) {
+        final Path reportsLogs = Paths.get("reports", "logs");
+        final Path logsDir = rootDirectory.toPath().resolve(reportsLogs);
+        if (Files.notExists(logsDir)) {
+            try {
+
+                Files.createDirectories(logsDir);
+            } catch (IOException e) {
+                throw new IllegalArgumentException(String.format("Could not created logs directory at %s", logsDir));
+            }
+        }
+        return logsDir.toFile();
+    }
+
     private void addEntry(PropertyEntry propertyEntry, GroupEntry groupEntry) {
         groupEntry.getPropertyEntries().add(propertyEntry);
     }
 
+    private Map<String, Map<String, ?>> getStats(Statistics statistics, Boolean decimal) {
+
+        Map<String, Map<String, ?>> stats = new LinkedHashMap<>();
+        if (statistics != null){
+            Map<String, Map<String, String>> networks = extractNetworksStats(statistics.getNetworks(), decimal);
+            Map<String, String> memory = extractMemoryStats(statistics.getMemoryStats(), decimal, "usage", "max_usage", "limit");
+            Map<String, String> blkio = extractIORW(statistics.getBlkioStats(), decimal);
+            stats.put("network", networks);
+            stats.put("memory", memory);
+            stats.put("block I/O", blkio);
+        }
+        return stats;
+    }
+
+    private Map<String, Map<String, String>> extractNetworksStats(Map<String, Object> map, boolean decimal) {
+        Map<String, Map<String, String>> nwStatsForEachNICAndTotal = new LinkedHashMap<>();
+        if (map != null) {
+            long rx_bytes = 0, tx_bytes = 0;
+
+            for (Map.Entry<String, Object> entry: map.entrySet()) {
+                Map<String, String> nwStats = new LinkedHashMap<>();
+                String adapterName = entry.getKey();
+                if (entry.getValue() instanceof LinkedHashMap) {
+
+                    Map<String, ?> adapter = (LinkedHashMap) entry.getValue();
+
+                    long rx_bytes_ = convertToLong(adapter.get("rx_bytes"));
```

Comment:
I will change name vars to `rxBytes` and `txBytes` to have a Java like code style.